### PR TITLE
Level1

### PR DESCRIPTION
**Pull request by NaveenSidhartha**
1. Added Curses view
2. Added copyright
3. Added Level 1 of Installer flow chat

### CODE REVIEW

File: configuration.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+        except ValueError:
+            ipv6_link_subnet = ipaddress.ip_network('::/126')
+            error_list.append('Test (1.2.15): Invalid "ipv6_link_subnet".It is NOT valid IP address subnet.')
+            result = False
+
+        if ipv6_link_subnet in ipv6_subnet:
+            error_list.append('Test (1.2.17): "ipv6_link_subnet" must NOT be within "ipv6_subnet".')
+            result = False
+
+    # ceph_monitors only for podnet
```
Comment (maria-walsh) [inline configuration.py:199]:
```suggestion
    # ceph_monitors only for an appliance that its blend includes the region flavour
```
Can you review this test please? Ceph monitors will only be in the config.json on an appliance if the blend includes region.

Also it is an optional field and can be null. The reason for this is, it is not a requirement for all regions to support Ceph

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 jinja2
 netaddr
 tabulate
+yaml
```
Comment (maria-walsh) [inline requirements.txt:4]: Where is this library utilised, I do not see it imported

**Review by maria-walsh — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (NaveenSidhartha) [reply requirements.txt:4]: Oh, it is required in the next push. gonna used in network.py. I didn't push the network.py changes as this branch is level1. 
Let me remove that.

Comment (NaveenSidhartha) [reply requirements.txt]: removed

**Review by maria-walsh — APPROVED** [review]

**Pull request merged**